readme.txt

```diff
@@ -231,6 +231,10 @@ Remember to always make a backup of your database and files before updating!
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Tweak - Added a Calendar Template section in the event display settings page. [ECP-1618]
```

Comment:
There is a lot of change here around usage of `gmdate` instead of `date`, we need to changelog that.

Reply:
Nte teh cnages only affect the sample dates on the display settings page.
...and quiets a VIP phpcs error 😉